web/themes/custom/pineapple/components/_heading.twig

```diff
@@ -0,0 +1,10 @@
+{% set heading_attributes = heading_attributes|default(create_attribute()) %}
+{% set heading_level = heading_level|default('3') %}
+
+{% if heading_level == '1' %}
+  <h1 {{ heading_attributes }}>{{ heading }}</h1>
```

Comment:
You should probably call the heading variable heading_text so that it's more descriptive